Classes/Domain/Model/ClassObject/ClassObject.php

```diff
@@ -413,16 +407,14 @@ public function setPropertyTag($propertyName, $tag)
     public function propertyExists($propertyName)
     {
         if (!is_array($this->methods)) {
-            $result = false;
-        } else {
-            if (in_array($propertyName, $this->getPropertyNames())) {
-                $result = true;
-            } else {
-                $result = false;
-            }
+            return false;
         }
 
-        return $result;
+        if (in_array($propertyName, $this->getPropertyNames())) {
+            return true;
+        }
+
+        return false;
```

Comment:
even better:
return is_array($this->methods) && in_array($propertyName, $this->getPropertyNames());